graylog2-web-interface/src/components/event-definitions/event-definition-form/field-value-providers/LookupTableFieldValueProviderFormContainer.tsx

```diff
@@ -41,40 +28,9 @@ type Props = {
   onChange: () => void,
 }
 
-const LookupTableFieldValueProviderFormContainer = ({ currentUser, ...otherProps }: Props) => {
-  const { data: fieldTypes } = useFieldTypes([], ALL_MESSAGES_TIMERANGE);
-  const lookupTables = useStore(LookupTablesStore);
-
-  useEffect(() => {
-    if (!isPermitted(currentUser.permissions, LOOKUP_PERMISSIONS)) {
-      return;
-    }
-
-    LookupTablesActions.searchPaginated(1, 0, undefined, false);
-  }, [currentUser.permissions]);
-
-  if (!isPermitted(currentUser.permissions, LOOKUP_PERMISSIONS)) {
-    return (
-      <Row>
-        <Col md={6} lg={5}>
-          <p>No Lookup Tables found.</p>
-        </Col>
-      </Row>
-    );
-  }
-
-  const isLoading = !fieldTypes || !lookupTables.tables;
-
-  if (isLoading) {
-    return <Spinner text="Loading Field Provider information..." />;
-  }
-
-  return (
-    <LookupTableFieldValueProviderForm allFieldTypes={fieldTypes}
-                                       lookupTables={lookupTables.tables}
-                                       {...otherProps} />
-  );
-};
+const LookupTableFieldValueProviderFormContainer = ({ ...otherProps }: Props) => (
+  <LookupTableFieldValueProviderForm {...otherProps} />
+);
 
 LookupTableFieldValueProviderFormContainer.propTypes = {
```

Comment:
Can't we remove the component now and just use `LookupTableFieldValueProviderForm` as a `formComponent` for the related plugin?